gamemodes/terrortown/gamemode/client/cl_equip.lua

```diff
@@ -27,8 +27,21 @@ local serverColsVar = GetConVar("ttt_bem_sv_cols")
 local serverRowsVar = GetConVar("ttt_bem_sv_rows")
 local serverSizeVar = GetConVar("ttt_bem_sv_size")
 
--- add favorites DB functions
-include("favorites_db.lua")
+-- setup database
+if not sql.TableExists("ttt_bem_fav") then
+	sql.Query("CREATE TABLE ttt_bem_fav (guid TEXT, role TEXT, weapon_id TEXT)")
+end
```

Comment:
this part needs to be replaced by a migration as well, but as this table was already created on many servers without the migration library I first need to implement something in the migration library to handle these kinds of situations